### DISCUSSION TITLE
Trying to use ffmpeg to stream rtsp video streaming to http

### DISCUSSION THREAD

**Discussion by helloworld20001000** [bluenviron/mediamtx#1677] [Questions]
Hi everyone who is reading this post! 
I'm trying to transcode the RTSP stream pulled by my configured RTSP proxy server to HTTP-stream with ffmpeg but when I execute command in Linux: $ ffmpeg -i rtsp://ip_proxy_server:8554/path/to/stream http://ip_proxy_server:8090/test.mp4
It returns the next error:
"Connection to tcp://localhost:8090 failed: Connection refused"
"http://localhost:8090/test.mp4: Connection refused"
I'm running the proxy server in a docker container based in CentOS 8 distribution. As subscription-manager is not enabled in docker containers using redhat OS I can't install VLC to stream RTSP video to be accessible through HTTP. I was able to install ffmpeg through git repositories and I read in different forums that it was posible to transcode RTSP stream to HTTP but I've tried it in so many different ways that I don't give with the proper solution to make it work. Does anyone have any good idea? Thanks before hand!

**Comment by helloworld20001000** [comment]
Hi! I've already worked it out. I just read the docs and enabled the HLS encoded in the RTSP server to transform the RTSP streaming to be accessible through HTTP protocol. Thank u!

**Reply by volkankoc** [reply]
> Hi! I've already worked it out. I just read the docs and enabled the HLS encoded in the RTSP server to transform the RTSP streaming to be accessible through HTTP protocol. Thank u!

I have same problem too. can you share yml file and html file (player url)

[a comment by an automated account is not shown]
